api/auth/signin.js

```diff
@@ -0,0 +1,4 @@
+async ({ login, password }) => {
```

Comment:
IIRC
We usually need `public: true` classifier for auth methods, otherwise impress/metacom will respond with 401.